Services/LeverPostingService.cs

```diff
@@ -101,9 +102,11 @@ private async Task<IList<ContentItem>> CreateUpdateAsync(IList<Posting> postings
             var postingContentItems = await GetAllAsync();
 
             // Remove old postings
+            _logger.LogInformation($"Lever: Remove content items from API");
             await RemoveAsync(contentManager, postingContentItems.Where(x => !postings.Any(y => y.Id == x.As<LeverPostingPart>().LeverId)).ToList());
 
             // Add/Update posings
+            _logger.LogInformation($"Lever: Add/Update content items from API");
```

Comment:
It would be useful to include the count of the number of postings in this log message.